### PR TITLE
Update Cluster Autoscaler entrypoint

### PR DESCRIPTION
**Pull request by MaciekPytel**
**What this PR does / why we need it**:
Update Cluster Autoscaler manifest file to use new shell wrapper instead of directly calling CA binary (the wrapper is already included in current CA image). 
Add params to improve logging.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by eparis** [comment]
/approve
@k8s-bot ok to test

**Comment by mwielgus** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by mwielgus** [comment]
@k8s-bot kops aws e2e test this

File: cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest [hunk 1]
```diff
@@ -27,9 +27,18 @@
...
+                    "--kubernetes=http://127.0.0.1:8080?inClusterConfig=f",
+                    "--v=4",
+                    "--stderrthreshold=info",
+                    "--write-status-configmap=true",
+                    "{{params}}"
+                ],
+                "env": [
+                    {
+                        "name": "LOG_OUTPUT",
+                        "value": "/var/log/cluster-autoscaler.log"
```
Comment (rata) [inline cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest]: just curious, can't this fill the container disk? What happens if this fills the disk?

Comment (mwielgus) [inline cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest]: This file is mounted from the machine disk. I guess that there is some log rotation policy for /var/logs on the master (but can doublecheck).

**Pull request merged**